src/wide-character.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+
+/**
+ * This file provides support for Unicode range U+1D400 to U+1D7FF,
+ * Mathematical Alphanumeric Symbols.
+ *
+ * Function wideCharacterFont takes a wide character as input and returns
+ * the font information necessary to render it properly.
+ */
+
+import type {Mode} from "./types";
+import ParseError from "./ParseError";
+
+/**
+ * Data below is from https://www.unicode.org/charts/PDF/U1D400.pdf
+ * That document sorts characters into groups by font type, say bold or italic.
+ *
+ * In the arrays below, each subarray consists three elements:
+ *      * The CSS class of that group when in math mode.
+ *      * The CSS class of that group when in text mode.
+ *      * The font name, so that KaTeX can get font metrics.
+ */
+
+const wideLatinLetterData: Array<[string, string, string]> = [
```

Comment:
It would be good to introduce some types like:
```
type MathCssClass = "mathbf" | "mathit" | ...;
type TextCssClass = "textbf" | "textit" | ...;
type FontName = "Main-Bold" | "MainItalic" | ...;
```
This will make it easier to pick up errors as people continue to work on this code.

Reply:
I'll work on it.

Reply:
I tried adding the types that you suggested. The result was an enormous number of flow errors. My flow skills are not strong.

So I have reverted to specifying a string in the most recent commit.